airflow/utils/email.py

```diff
@@ -49,10 +49,17 @@ def send_email(
     """Send email using backend specified in EMAIL_BACKEND."""
     backend = conf.getimport('email', 'EMAIL_BACKEND')
     backend_conn_id = conn_id or conf.get("email", "EMAIL_CONN_ID")
+    from_email = (
+        conf.get('email', 'email_from_email') if conf.has_option('email', 'email_from_email') else None
+    )
+    from_name = conf.get('email', 'email_from_name') if conf.has_option('email', 'email_from_name') else None
```

Comment:
IIRC the previous version added support for some config options under `ses` and `sendgrid` (or something). Are those still respected?

Reply:
No, those config options were only introduced in the previous version of PR. When refactoring I thought there is no reason to have sender config (email & name) in each backend separately.

Reply:
I see. Since this is essentially the “last resort” I think using one single value (pair) makes sense.